### PR TITLE
segmented-control: adjust colors for classic bright

### PR DESCRIPTION
**Pull request by flootr**
#### Changes proposed in this Pull Request

* Adjust `<SegmentedControl />` colors for _Classic Bright_

#### Testing instructions

* design: are those colors appropriate? do we need to adjust them?
* code: make sure there's a proper replacement for any 

segmented-control

**Note:** this merges into `features/color-refresh-2019`

### CODE REVIEW

File: client/components/segmented-control/style.scss [hunk 1]
```diff
@@ -80,31 +80,31 @@
...
+				border-left-color: var( --color-primary );
 			}
 		}
 	}
 
-	.segmented-control__link:focus {
-		background-color: lighten( $blue-light, 22.5% );
-	}
+	// .segmented-control__link:focus {
+	// 	background-color: lighten( $blue-light, 22.5% );
```
Comment (flootr) [inline client/components/segmented-control/style.scss]: I'm not sure about a replacement for these color and would love some feedback @drw158 on what color would work?

Comment (drw158) [reply client/components/segmented-control/style.scss]: The equivalent color of `lighten( $blue-light, 22.5% )` is roughly `blue-0`, but it's so desaturated, I think we can use `var( --color-neutral-0` here.

**Comment by drw158** [comment]
Looks great on design 👍

**Comment by flootr** [comment]
It's two different states for the selected item. The one is focused (which it looses if you click outside) and the other is not focused.

**Comment by frontdevde** [comment]
If that's the intended look for the focus state, then that's great. Just thought I'd double check.

**Comment by flootr** [comment]
@frontdevde thanks for checking. @drw158 let me know if you want any other adjustments and I'm happy to follow up with another PR

**Pull request merged**